devutils/lib/bash/bazel.bash

```diff
@@ -39,7 +39,11 @@ Bazel::TestTargets () {
 
     while read target
     do
-        bazel test --ignore_unsupported_sandboxing $target || return 1
+        $made_tests || Log::Notice "testing associated targets (this might take a while)"
```

Comment:
Even thought it is not mandatory, one would prefer use `test $myVar` or `( $myVar )`